spec/features/preregister_for_annuities_spec.rb

```diff
@@ -0,0 +1,59 @@
+require 'spec_helper'
+
+RSpec.describe 'Pre-Register for an annuities appointment', type: :feature do
+  let(:test_api_class) do
+    Class.new do
+      attr_reader :last_called_with
+
+      def create_ticket(args)
+        @last_called_with = args
+      end
+    end
+  end
+  let(:test_api) { test_api_class.new }
+
+  around do |ex|
+    begin
+      real_api = ZenDesk.api
+      ZenDesk.api = test_api
+      ex.run
+    ensure
+      ZenDesk.api = real_api
+    end
+  end
+
+  it 'preregister for an appointment' do
+    when_i_complete_annuities_preregistration
+    then_my_details_have_stored_for_future_contact
+  end
+
+  def when_i_complete_annuities_preregistration # rubocop:disable Metrics/MethodLength
+    visit '/register-for-annuities'
+
+    fill_in 'First name', with: 'Rick'
```

Comment:
We have not used SitePrism in any of the Rspec feature tests, so I have kept consistent this that.

I meant to check the reason but didn't get a chance. @benbarnett do you know why we haven't used SitePrism in the rspec tests historically?